packages/adapters/base/src/entities/Schema.ts

```diff
@@ -0,0 +1,28 @@
+import {
```

Comment:
I guess this explains my previous Q. Looks like we're following the Prisma API (as of 2024)

Reply:
Pretty much at this time we are porting all the prisma calls with a direct sdk call using the conventions of `packages/schemas` & `pakages/graphql`.

Reply:
As prisma is the only storage adapter at this point, however, the api will mold and change as we add more adapters. 

So this work is essantly just a big refactor to set us up for futuer adapt